src/handlers/autolabel.rs

```diff
@@ -60,6 +60,14 @@ impl Handler for AutolabelHandler {
                     }
                 }
             }
+            if e.action == github::IssuesAction::Closed {
+                let labels = event.issue().unwrap().labels();
+                if let Some(x) = labels.clone().into_iter().position(|x| x.name == "I-prioritize") {
```

Comment:
Something about clone().into_iter feels off here - can this be just iter() with perhaps a few derefs in the position closure?